src/StringUtils.cpp

```diff
@@ -6,6 +6,8 @@
 #include "Globals.h"
 
 #include "fmt/printf.h"
+#include "json/json.h"
```

Comment:
The compiler complains about this line.

From what I can tell, the StringUtils file is used in other subprojects that don't have the json library added to them. You need to add the jsoncpp dependency to GrownBiomeGenVisualiser.

Details: https://builds.cuberite.org/blue/organizations/jenkins/cuberite/detail/PR-5508/1/pipeline